spec/spec_helper.rb

```diff
@@ -3,6 +3,8 @@
 
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'circuitry'
+require 'circuitry/cli'
+require 'circuitry/provisioning'
```

Comment:
`circuitry/cli` already requires `circuitry/provisioning`, so you don't need to explicitly require provisioning here.